packages/compiler/src/lib/decorators.ts

```diff
@@ -1229,3 +1229,43 @@ export function getDiscriminator(program: Program, entity: Type): Discriminator
 export function getDiscriminatedTypes(program: Program): [Model | Union, Discriminator][] {
   return [...program.stateMap(discriminatorKey).entries()] as any;
 }
+
+const parameterVisibilityKey = createStateSymbol("parameterVisibility");
+
+export function $parameterVisibility(
+  context: DecoratorContext,
+  entity: Operation,
+  ...visibilities: string[]
+) {
+  validateDecoratorUniqueOnNode(context, entity, $parameterVisibility);
+  context.program.stateMap(parameterVisibilityKey).set(entity, visibilities);
+}
+
+/**
+ * Returns the visibilities of the parameters of the given operation, if provided with `@parameterVisibility`.
+ *
+ * @see $parameterVisibility
```

Comment:
```suggestion
 * @see {@link $parameterVisibility}
```

i think is the right syntax so it links it in the docs

Reply:
I don't see any examples in code where we use `@see` to reference any internal components with a link, only external documentation, so I just removed these.